src/SAWScript/Crucible/LLVM/ResolveSetupValue.hs

```diff
@@ -405,10 +430,31 @@ resolveSAWSymBV cc w tm =
              _ -> return Nothing
      case mx of
        Just x  -> W4.bvLit sym w (BV.mkBV w x)
-       Nothing -> bindSAWTerm sym st (W4.BaseBVRepr w) tm
+       Nothing
+         | doW4Eval ?w4EvalTactic ->
```

Comment:
There's quite a bit of code shared between `resolveSAWSymBV` here and `resolveSAWPred`. Can this be factored out a bit more to avoid code duplication?

Reply:
I want to keep this pull request minimal. In general, I agree that the code would benefit from engineering improvements, and this case doesn't even make priority list.